packages/zcli-apps/src/commands/apps/create.ts

```diff
@@ -46,7 +48,11 @@ export default class Create extends Command {
       try {
         const { app_id }: any = await getUploadJobStatus(job_id, appPath)
         cli.action.stop('Deployed')
-        const settings = manifest.parameters ? await promptAndGetSettings(manifest.parameters) : {}
+
+        const allConfigs = getAllConfigs(appPath)
+        const configParams = allConfigs?.parameters || {} // if there are no parameters in the config, just attach an empty object
+
+        const settings = manifest.parameters ? await getAppSettings(manifest, configParams) : {}
```

Comment:
Have you covered the case when `configParams` is `{}` here?

Reply:
Just checked, yes so we have two apps one with no config params and one with params. Functional tests run against both, so they are covered.